GOVERNANCE.md

```diff
@@ -21,7 +21,7 @@ Anybody is free to use the core FOSSBilling software within the terms of the [li
 
 We really appreciate user contributions but if you just want to download, install, and go, then you have the complete freedom to do that.
 
-Third-party extensions to FOSSBilling are also welcomed, it would be appreciated if these were discussed and informed by the community, but this is not mandatory. In these cases please do be aware of the restrictions attached to usage of the [FOSSBilling name and branding](https://github.com/FOSSBilling/branding#readme). 
+Third-party modules and integrations for FOSSBilling are welcomed. We would apprciate if your third-party work was announced to the community so that it can be added to the list of known working extensions, however this is not mandatory. In all cases, please be aware of the restrictions attached to usage of the [FOSSBilling name and branding](https://github.com/FOSSBilling/branding#readme), and approach us for permission if you wish to use the branding in any way other than explicitly listed. 
```

Comment:
Nice, seems very clear! Just a small typo on "appreciate" :D

Reply:
Fixed, I appreciate you noticing :)